### PR TITLE
Fix/search result

### PR DESCRIPTION
**Pull request by eouin**
Fix the header with new design/layout
Fix the search bar

Goran did the ui review and he's happy with it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
         "react-dom": "^16.8.2",
         "react-dropdown": "^1.6.4",
         "react-helmet": "^5.2.0",
+        "react-inlinesvg": "^0.8.4",
         "react-redux": "^6.0.0",
```
Comment (ckoutsiaris) [inline package.json]: Do we need the redux package?

File: src/components/Layout/Layout.js [hunk 2]
```diff
@@ -35,13 +37,21 @@ class Layout extends React.Component {
...
       const values = queryString.parse(this.props.location.search);
+
       if (values.q) {
         this.toggleSearch(true);
         this.setState({ searchValue: values.q });
       } else {
         this.toggleSearch(false);
         this.setState({ searchValue: '' });
       }
+      const fromUrl = this.props.location.state.fromUrl || '/';
```
Comment (ckoutsiaris) [inline src/components/Layout/Layout.js]: Can this be undefined? If no then the next line will always be true?

Comment (eouin) [reply src/components/Layout/Layout.js]: yep it can be 'undefined'

File: package.json (hunk 1 above)
Comment (eouin) [reply package.json]: It's not yet used, I'm planning to have a go at it next as this state management are becoming crazy.

**Pull request merged**